### PR TITLE
Milestone #4: Refactor Class Components to Function Components Using Hooks

### PR DESCRIPTION
**Pull request by Anjacodes**
In this milestone, I: 

* Changed all class components to function components
* Added state using hooks

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,14 +1,11 @@
...
-    );
-  }
-}
+export const App = () => { // eslint-disable-line
+  return (
+    <div>
+      <Calculator />
+    </div>
+  );
+};
```
Comment (er-contreras) [inline src/App.js]: Great work changing from class-based to hooks but there is a simple thing you can do with that eslint comment instead you just can fix it like this. Just remove the curly braces to get clean your code.

 `export const App = () => (
    <div>
     <Calculator />
    </div>
 );
`

File: src/components/Button.js [hunk 2]
```diff
@@ -1,23 +1,21 @@
...
-    return (
-      <button onClick={this.props.handleClick} className={this.props.class} type="button">{this.props.text}</button> // eslint-disable-line
-    );
-  }
-}
+export const Button = ({ handleClick, addClass, text }) => { // eslint-disable-line
+  return (
+    <button onClick={handleClick} className={addClass} type="button">{text}</button>
+  );
+};
```
Comment (er-contreras) [inline src/components/Button.js]: - It would be great to remove the eslint-disable-line comment to get more clean your code.

**Review by er-contreras — CHANGES_REQUESTED** [review]
Hi @Anjacodes , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

almost there

## Required Changes ♻️

- [ ] In your readme file we got a broken link in the math magicians preview.
- [ ] Also It would be great to have the MIT link only instead of the whole text.
- [ ] If you don't have live demo link yet avoid that in the readme file

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DeliceLydia — APPROVED** [review]
# Congratulations🎉🎉

## Hi @Anjacodes, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**